app/config.py

```diff
@@ -503,3 +503,14 @@ def setup_nameservers():
     raise RuntimeError(
         "Please define RECOVERY_CODE_HMAC_SECRET in your configuration with a random string at least 16 chars long"
     )
+
+# run over all reverse alias for an alias and replace them with sender address
+ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT = (
+    "ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT" in os.environ
+)
+
+if ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT:
+    # max number of reverse alias that can be replaced
+    MAX_NB_REVERSE_ALIAS_REPLACEMENT = int(
+        os.environ["MAX_NB_REVERSE_ALIAS_REPLACEMENT"]
+    )
```

Comment:
```suggestion
    MAX_NB_REVERSE_ALIAS_REPLACEMENT = int(
        os.environ.get("MAX_NB_REVERSE_ALIAS_REPLACEMENT",5)
    )
```

Reply:
I don't put a default value here to avoid us from forgetting to set `MAX_NB_REVERSE_ALIAS_REPLACEMENT` if `ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT` is enabled actually :)

Reply:
Better to add an error then, but avoid stopping production from running if we make a config mistake.

Reply:
Not sure to understand, not setting it will trigger an error and stop the container from starting which is what we wanted right?

Reply:
Yeah. I mean instead of having the container from starting. Show an error but allow the container to start to avoid breaking production.

Reply:
prod shouldn't be broken as the existing containers will continue to run right. I just run a test with `ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT` set but not `MAX_NB_REVERSE_ALIAS_REPLACEMENT` and the existing container continues to run.

Reply:
What? How does it run? It should fail becase that should throw an exception and abort.

Reply:
If `ENABLE_ALL_REVERSE_ALIAS_REPLACEMENT` is set but not `MAX_NB_REVERSE_ALIAS_REPLACEMENT` that throws an `KeyException` and the app won't start.